scripts/ci.sh

```diff
@@ -2,30 +2,41 @@
 
 set -euo pipefail
 
+dependency_hash=$( (cat package.yaml; cat snapshot.yaml) | sha256sum | cut -d ' ' -f 1)
+
+remote="gs://radicle-build-cache/v2"
+
+local_cache_archive="stack-root.tar.gz"
+remote_cache_master="$remote/stack-root-master.tar.gz"
```

Comment:
I’m not sure I understand the reason for two cache bundles - why is master different?

Reply:
We have master caches to make changes to the dependencies still be fast: Assume we add a dependency and build the code for the first time. No cache exists for the calculated key. Without the master cache we would rebuild the code from scratch. With the master cache we only need to build the added dependencies.

I made some changes to the `save-cache` function so that the overhead adding that cache should be minimal. We just copy the existing hashed cache in the cloud.